ansible_runner/utils/capacity.py

```diff
@@ -24,24 +24,21 @@ def get_mem_in_bytes():
         return error
 
 
-def get_uuid():
```

Comment:
Do we need to keep `get_uuid()` around for backwards compatibility?

Reply:
It probably wouldn't hurt. We can just deprecate it. Deleting it entirely is a bit abrupt since it's a public function.

Reply:
I don't think so, it was never shipped, just landed a month ago https://github.com/ansible/ansible-runner/commit/49353d2d1d6a4fbb9ed47c3ca9a2b6e82f392e41

Reply:
Sweet!